web/profiles/webspark/webspark/webspark.install

```diff
@@ -447,3 +447,13 @@ function webspark_update_10001(&$sandbox) {
     Drupal::service('module_installer')->install(['layout_builder_usage_reports']);
   }
 }
+
+/**
+ * WS2-1565 - CK Editor - Remove H1 capabilities for non hero blocks.
+ */
+function webspark_update_10002(&$sandbox) {
+  \Drupal::state()->set('configuration_locked', FALSE);
+  \Drupal::service('webspark.config_manager')->updateConfigFile('editor.editor.basic_html');
+  \Drupal::service('webspark.config_manager')->updateConfigFile('filter.format.basic_html');
```

Comment:
The input format settings aren't locked in Webspark, so doing this update by importing the default editor configs could result in overriding sites' customizations of the setting.

As an alternative approach, could we load the active settings, make the change to the H1 setting on that object, and then resave the active settings?

If we cannot automate that in an update hook, then I think we will need to document this as a manual change site owners will need to make when the apply the next update.